crates/proof-of-sql/src/proof_primitive/dynamic_dory_and_hyrax_common_utils/standard_basis_helper.rs

```diff
@@ -97,33 +138,35 @@ fn build_partial_second_half_standard_basis_vecs(
     }
 }
 
-#[allow(dead_code)]
-pub(super) fn fold_dynamic_standard_basis_tensors(
-    point: &[F],
-    alphas: &[F],
-    alpha_invs: &[F],
-) -> (F, F) {
+pub(crate) fn fold_dynamic_standard_basis_tensors<S: Scalar, I: Into<S> + From<S> + Copy>(
```

Comment:
This needs a doc comment please

Reply:
This is not a new function. I'm not really sure what the point of this is.